integration-test/src/test/resources/trino-ci-testset/testsets/jdbc-postgresql/00007_varchar.txt

```diff
@@ -0,0 +1,47 @@
+CREATE SCHEMA
+
+USE
+
+CREATE TABLE
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb01 (
+   id integer,
+   name char(20)
+)
+COMMENT ''"
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb02 (
+   id integer,
+   name char(65536)
+)
+COMMENT ''"
+
+<QUERY_FAILED> Unknown type 'char(65537)' for column 'name'
+
+CREATE TABLE
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb04 (
+   id integer,
+   name varchar(250)
+)
+COMMENT ''"
+
+CREATE TABLE
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb05 (
+   id integer,
+   name varchar(10485760)
+)
+COMMENT ''"
+
+<QUERY_FAILED> Failed to create table.
+
```

Comment:
Missing  line 25 command output, Does it still work with subsequent commands?
```
SHOW CREATE TABLE "test.gt_postgresql".varchar_db1.tb06;
```

Reply:
OK